tests/installation/disable_grub_timeout.pm

```diff
@@ -48,14 +48,11 @@ sub run {
     save_screenshot;    # needed because shortcuts change with the navigation
 
     # Select bootloader options tab
-    if ((is_sle && !sle_version_at_least('12-SP2')) || (is_leap && !leap_version_at_least('15'))) {
-        $cmd{bootloader} = 'alt-t';    # older versions
-    }
-    elsif (is_leap) {
+    if (is_leap && check_var('VERSION', '15')) {
```

Comment:
no, don't use a version specific check. We don't want to revisit this for '15.0' or '15-SP1'. What's wrong with the old code?

Reply:
The problem of the old is that there is a too complicated if-else-statements when the only different version that doesn't match the pattern (UEFI->t, non-UEFI->r) is Leap 15. So I simplified the to a single if-statement.

Would it be better if I use:

```
if (is_leap && leap_version_at_least('15')
```